bee-message/src/message.rs

```diff
@@ -148,6 +148,22 @@ impl Message {
     pub fn into_parents(self) -> Parents {
         self.parents
     }
+
+    /// Unpacks a [`Message`] from a sequence of bytes doing syntactical checks and verifying that
+    /// there are no traling bytes in the secuence.
```

Comment:
```suggestion
    /// there are no trailing bytes in the sequence.
```
Give typos no chance!